.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -158,6 +158,12 @@ jobs:
         timeout-minutes: 15
         run: ./.github/scripts/run_e2e_tests.sh
 
+      - name: Print debug if failed
+        if: ${{ failure() }}
+        run: |
+          cd bin/cliain
+          cargo run -- --seed //Alice debug-storage
```

Comment:
In general, I'm not a fan of putting any seeds anywhere, but here is justified as it is a known developer account for a  Substrate chain run in dev mode :)